src/rapidfuzz/process.pyi

```diff
@@ -16,110 +16,103 @@ from typing import (
 from rapidfuzz.fuzz import WRatio, ratio
 
 _StringType = Sequence[Hashable]
-_S1 = TypeVar("_S1")
-_S2 = TypeVar("_S2")
+_StringType1 = TypeVar("_StringType1", bound=Sequence[Hashable])
+_StringType2 = TypeVar("_StringType2", bound=Sequence[Hashable])
 _KeyType = TypeVar("_KeyType")
-_ResultType = int | float
+_ResultType = TypeVar("_ResultType")
```

Comment:
shouldn't this be `TypeVar("_ResultType", int, float)`?

Reply:
Same explanation of above